src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int RADIUS = 10;
+    public static final int FIGURE_QUANTITY = 5;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        Figure randomFigure;
+        String randomColor = Colors.values()[random.nextInt(Colors.values().length)].toString();
+        int randomNumber = random.nextInt(FIGURE_QUANTITY - 1);
+        switch (randomNumber) {
+            case 0:
+                randomFigure = new Circle(randomColor, random.nextDouble());
+                break;
+            case 1:
+                randomFigure = new Square(randomColor, random.nextDouble());
+                break;
+            case 2:
+                randomFigure = new IsoscelesTrapezoid(randomColor,
+                        random.nextDouble(), random.nextDouble(), random.nextDouble());
+                break;
+            case 3:
+                randomFigure = new Rectangle(randomColor, random.nextDouble(), random.nextDouble());
+                break;
+            case 4:
+                randomFigure = new RightTriangle(randomColor,
+                        random.nextDouble(), random.nextDouble());
+                break;
+            default:
+                randomFigure = null;
+        }
```

Comment:
it's a bad practice to return null, let's use default case instead of case 4 to return one of the random figures. But don't forget to decrease FIGURE_QUANTITY by 1